examples/core/rgcn/hetero_rgcn.py

```diff
@@ -0,0 +1,647 @@
+"""
+This script, `hetero_rgcn.py`, trains and tests a Relational Graph
+Convolutional Network (R-GCN) model for node classification on the
+Open Graph Benchmark (OGB) dataset "ogbn-mag". For more details on
+"ogbn-mag", please refer to the OGB website:
+(https://ogb.stanford.edu/docs/linkprop/)
+
+Paper [Modeling Relational Data with Graph Convolutional Networks]
+(https://arxiv.org/abs/1703.06103).
+
+
+This flowchart describes the main functional sequence of the provided example.
```

Comment:
Is it possible to follow the same flow with https://github.com/dmlc/dgl/blob/master/examples/sampling/node_classification.py as much as possible?

Reply:
ok